File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/PromptUpdateForm.kt
package com.sdkwork.app

data class PromptUpdateForm(
    val title: String? = null,
    val content: String? = null,
    val description: String? = null,
    val cateId: Int? = null,
    val enabled: Boolean? = null,
    val sort: Int? = null,
    val parameters: Map<String, Any>? = null,
    val model: String? = null,
    val isPublic: Boolean? = null
)

File: sdkwork-app-sdk-go/types/prompt_update_form.go
package types

// 更新提示语请求
type PromptUpdateForm struct {
	Title string `json:"title"`
	Content string `json:"content"`
	Description string `json:"description"`
	CateId int `json:"cateId"`
	Enabled bool `json:"enabled"`
	Sort int `json:"sort"`
	Parameters map[string]interface{} `json:"parameters"`
	Model string `json:"model"`
	IsPublic bool `json:"isPublic"`
}
